### ISSUE TITLE
Accessibility issue on create collection - nav does not have accessible name

### ISSUE THREAD

**Issue by astridu** [sul-dlss/happy-heron#1041]
![image](https://user-images.githubusercontent.com/42216982/106979281-de34b300-6712-11eb-9124-5b73b9bfebda.png)
